src/System.Management.Automation/commands/utility/FormatAndOutput/common/BaseOutputtingCommand.cs

```diff
@@ -921,6 +921,18 @@ internal override void Initialize()
                 }
 
                 int columnsOnTheScreen = this.InnerCommand._lo.ColumnNumber;
+                // for tables, we want to make sure we don't end up adding padding to int.MaxValue and default to 120 columns instead
```

Comment:
Can we rephrase it for easy reading? 

And why 120? Can we use current windows width?

Reply:
Let me try to add a bit more detail to make it easier to understand.  The code tries to use window width, but if it can't be read (like implicit remoting where there's no console window), we have to default to something.  120 was the original default without my changes.